test/sql/stringfunctions_sql_test.cpp

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// Compression Testing Script
+//===----------------------------------------------------------------------===//
+#include <memory>
+#include <math.h>
+#include "sql/testing_sql_util.h"
+#include "catalog/catalog.h"
+#include "storage/database.h"
+#include "storage/data_table.h"
+#include "concurrency/testing_transaction_util.h"
+#include "common/harness.h"
+#include "executor/create_executor.h"
+#include "planner/create_plan.h"
+
+namespace peloton {
+namespace test {
+class StringFunctionTest : public PelotonTest {};
+/*
+The following test inserts 500 tuples in the datatable.
+Each tuple inserted is of the form (i, i * 'a' ), where i belongs to [0,500)
+We then perform a sequential scan on the table and retrieve the id and length
+of the second column.
+*/
+TEST_F(StringFunctionTest, LengthTest) {
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, txn);
+  catalog::Catalog::GetInstance()->Bootstrap();
+  txn_manager.CommitTransaction(txn);
+  // Create a t
+  txn = txn_manager.BeginTransaction();
+
+  TestingSQLUtil::ExecuteSQLQuery(
+      "CREATE TABLE foo(id integer, name varchar(500));");
+  int i;
+  std::string s = "'", t = "'";
+  for (i = 0; i < 500; i++) {
```

Comment:
Is there any reason that we have 500 tuples? Otherwise, let's pick something like 32, or less. We are not testing the tuple insertion here, so we don't need to waste our time running the test cases. It is relatively small, but they all add up.

Reply:
Fix that part. Thanks.